tests/integration/ember-engines-test.js

```diff
@@ -0,0 +1,50 @@
+import { module, test } from 'qunit';
+import {
+  setupContext,
+  setupRenderingContext,
+  teardownContext,
+  teardownRenderingContext,
+  render,
+} from '@ember/test-helpers';
+import engineResolverFor from 'ember-engines/test-support/engine-resolver-for';
+import hbs from 'htmlbars-inline-precompile';
+
+const modulePrefix = 'eager-engine';
+const resolver = engineResolverFor(modulePrefix);
+
+module('setupRenderingContext for "ember-engines"', function (hooks) {
+  hooks.beforeEach(async function () {
+    await setupContext(this, { resolver });
+    // this.owner now is an engine.
+    this.engine = this.owner;
```

Comment:
Hmm, can you actually make it a different owner (just aliasing them doesn't allow us to confirm that things actually function)? You can use the `build-owner.ts` file to make a different owner instance.

Reply:
@rwjblue `buildOwner` require an application and resolver as params, how can I create a `engine` instance with this file? 

are you proposing to create a `buildEngine.ts` or something like this?

Reply:
I think our `build-owner.ts` doesn't have the same API to support build engines as done in ember.js repo - https://github.com/emberjs/ember.js/blob/0554471ba64d9ade1dd1c111e53803fa5f7a97b6/packages/internal-test-helpers/lib/build-owner.js